scripts/obtain/dotnet-install.sh

```diff
@@ -534,17 +534,18 @@ get_absolute_path() {
 copy_files_or_dirs_from_list() {
     eval $invocation
 
-    local root_path=$(remove_trailing_slash $1)
-    local out_path=$(remove_trailing_slash $2)
-    local override=$3
-    local override_switch=$(if [ "$override" = false ]; then printf -- "-n"; fi)
+    local root_path="$(remove_trailing_slash "$1")"
+    local out_path="$(remove_trailing_slash "$2")"
+    local override="$3"
+    local override_switch="$(if [ "$override" = false ]; then printf -- "-n"; fi)"
```

Comment:
This line looked fine as it was; the resultant "-n" was quoted, I think that's enough...

Reply:
I agree, but I figured presenting quoted "$()" as a good style might also be warranted. Would you prefer the quotes here removed?

Reply:
Yes, please.